net.ankiweb.Anki.json

```diff
@@ -64,7 +64,8 @@
       "buildsystem": "simple",
       "build-commands": [ 
         "python3 configure.py --assume-shared --confirm-license --no-designer-plugin --no-qml-plugin --sysroot=/app -w --sip=/app/bin/sip --sip-incdir=/app/include -verbose QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.5m/' QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.5m/'", 
-        "make -j3", "make install DESTDIR=/app"
+        "make",
```

Comment:
Well, `make` just defaults to a single job. Do `make -j \`nproc\`` to use available threads.